### PR TITLE
fix(frontend): change cursor style over counter badge tm-395

### PR DESCRIPTION
**Pull request by Sytnikovakatya**

### CODE REVIEW

**Review by Oleksandra-Masiuk — APPROVED** [review]

**Review by GEOFARL — APPROVED** [review]

File: apps/frontend/src/libs/components/header/styles.module.css [hunk 1]
```diff
@@ -55,6 +55,10 @@
 	outline: 3px solid var(--main-white);
 }
 
+.counter:hover {
+	cursor: default;
```
Comment (ShadowOfTheSpace) [inline apps/frontend/src/libs/components/header/styles.module.css:59]: Is it need to be default or pointer?

Comment (Sytnikovakatya) [reply apps/frontend/src/libs/components/header/styles.module.css:59]: We can't click on this counter.  That's why I use default. Just to change text cursor that we had on arrow

Comment (Oleksandra-Masiuk) [reply apps/frontend/src/libs/components/header/styles.module.css:59]: @Sytnikovakatya please ask QA what behavior is expected in this case

Comment (Sytnikovakatya) [reply apps/frontend/src/libs/components/header/styles.module.css:59]: Yes, I asked QA. It can be default

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**